.github/workflows/release.yml

```
@@ -175,27 +175,113 @@ jobs:
+          
+          **Full Changelog**: https://github.com/tienld-0801/tilokit/compare/v0.1.0-dev...$VERSION
+          FALLBACK_EOF
           fi
           
-          # Save release notes to output and file
+          # Save release notes to output
           echo "RELEASE_NOTES<<EOF" >> $GITHUB_OUTPUT
-          echo "$RELEASE_NOTES" >> $GITHUB_OUTPUT
+          cat release_notes.md >> $GITHUB_OUTPUT
           echo "EOF" >> $GITHUB_OUTPUT
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)